frontend/apps/marketing/src/app/[locale]/layout.tsx

```diff
@@ -41,6 +42,8 @@ export default async function Layout({
           {children}
         </StatsigProvider>
       </OneTrustProvider>
+
+      {brand === Brand.CODE_DOT_ORG && <CDOJsonLd />}
```

Comment:
Just to make things a bit more general, could we make a general component?

`<JsonLd brand={brand} />`